sklearn/gaussian_process/correlation_models.py

```diff
@@ -11,9 +11,10 @@
 
 
 import numpy as np
+import warnings
 
 
-def absolute_exponential(theta, d):
+def absolute_exponential(theta, dx=None, d=None):
```

Comment:
Sorry, is this PR still active or superseded by #4023?
And I'm really not sure what his happening here. How is this change related to mismatching docstrings?